decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -127,9 +136,9 @@ def remove_sprockets_requirement
         gsub_file "config/environments/production.rb", /config\.assets.*$/, ""
       end
 
-      def copy_migrations
-        rails "decidim:choose_target_plugins", "railties:install:migrations"
-        recreate_db if options[:recreate_db]
+      def add_sprockets_configuration
+        # Create an empty file so Sprockets 4 do not complain
```

Comment:
I think this is one of my most controversial decisions :) 
As I want to have a really smooth upgrade process (and review), I don't want to ditch `rails assets:precompile`. 

I see that with #8595, you decided to ditch it and replace it with `rails webpacker:compile`. Maybe we could have the both of the two worlds, by:

1. Removing sprockets-rails 
2. Removing this `manifest.js` hack
3. Adding a `rails assets:precompile` letting users know that's deprecated and actually calling `webpacker:compile`.

Does this make sense? I'm thinking in how to make it easier/with less friction to implementers when upgrading.